test/validation/api/classification/odp_classification_test_pmr.c

```diff
@@ -2237,7 +2242,17 @@ static void test_pmr_series(const int num_udp)
 	pmr_param.val_sz      = sizeof(dst_addr_be);
 	pmr_param.offset      = 0;
 
-	pmr_ip = odp_cls_pmr_create(&pmr_param, 1, default_cos, cos_ip);
+	if (marking) {
```

Comment:
It would be nicer to have the repeating code put in a function that would take the "mark" and "marking" parameters in addition to the existing stuff. The test function is already quite long.

Reply:
I'd keep this code flat, since a generic function for IP/UDP PMR create would save only 10 lines. Also, flat validation test code is easier to use when debugging implementation as asserts/printfs can be inserted directly to the failing branch.